functions/concepts/termination.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/* eslint-disable */
+
+exports.nodeTermination = async (event, context, callback) => {
```

Comment:
We cannot have both `async` and `callback` in the same function. The Functions Framework won't resolve this correctly.

Either write:

```js
exports.nodeTermination = async (event, context) => {
```

or

```js
exports.nodeTermination = (event, context, callback) => {
```

Reply:
To be clear, this function is not actually meant to be run. (It's purely there to serve as a "wrapper" for the lines inside.)

(I can add a comment to that effect, if you want.)

Reply:
Just remove `callback` or remove `async`. If we need both, we need 2 style of functions.

Reply:
Done.